core/src/main/scala/kafka/server/ClientRequestQuotaManager.scala

```diff
@@ -28,13 +28,12 @@ import org.apache.kafka.server.quota.ClientQuotaCallback
 import scala.jdk.CollectionConverters._
 
 object ClientRequestQuotaManager {
-  val QuotaRequestPercentDefault = Int.MaxValue.toDouble
```

Comment:
please investigate why do we have this unused variable. Asking because it could be a bug where we want to use this but are not using this,

Reply:
The const was defining the default value for quota request percent: Before #10025 this was done in the DynamicConfig.scala class.

After #10025, we stopped using by just hardcoding the value: https://github.com/mumrah/kafka/blob/a9ede6eaedc1cdef2757bd2f03de1cb0efe170f8/clients/src/main/java/org/apache/kafka/common/config/internals/QuotaConfigs.java#L62

Reply:
ack. thank you.